projects/angular/src/forms/styles/_radio.clarity.scss

```diff
@@ -67,7 +67,7 @@
       //background-color around the dot
       @include css-var(
         box-shadow,
-        $clr-forms-radio-disabled-shadow,
+        clr-forms-radio-disabled-shadow,
```

Comment:
Is it a CSS Variable instead of a sass variable without the $? weird, not sure how I never noticed that before

Reply:
This is just the name of a css variable instead of the value of a sass variable. It's kinda confusing.